components/Layout/layout.module.scss

```diff
@@ -24,6 +24,7 @@
     }
 }
 .wrapper {
-    width: 80%;
+    //width: 80%;
+    width: 100%; //changed for signin page
```

Comment:
Instead of changing this to 100%, I think we should just disable the wrapper on this page. I had to do this for the Initial View component, so in the `components/Layout/index.tsx`, I added a prop called `initialView` that disables the wrapper and adds the IntialView component to the layout. We could adapt this for the future by adding another prop called something like `disableWrapper,` and when it's set to true, the wrapper is disabled. You could add something like this to the layout component after you add the prop:
```
{options?.wrapperDisabled && (
<section>
  {childen}
</section>
)}
```

Reply:
That's a good idea! I went ahead and implemented what you suggested. It should be pushed now